Unity-Technologies/ScriptableRenderPipeline
# New version of fog test

### Purpose of this PR
Change the fog test scene to render the different fog types with the different transparent blending types.
![1204_Lit_Fog](https://github.com/Unity-Technologies/ScriptableRenderPipeline/blob/e9d1a2daac9eca414abe308db17d759f3874f31d/TestProjects/HDRP_Tests/Assets/ReferenceImages/Linear/WindowsEditor/Direct3D11/1204_Lit_Fog.png?raw=true)

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?

**Manual Tests**: What did you do?

**Automated Tests**: What did you setup? (Add a screenshot or the reference image of the test please)
See higher.

---
### Overall Product Risks
**Technical Risk**: Low.

**Halo Effect**: None.
